lib/ansiblelint/rules/MetaTagValidRule.py

```diff
@@ -1,6 +1,8 @@
+#!/usr/bin/env python
```

Comment:
Why are you making it executable? I don't think that adding more way to do the same thing is a good idea.